src/navigation/ActivationStack.tsx

```diff
@@ -24,35 +23,42 @@ type ActivationStackParams = {
 
 const Stack = createStackNavigator<ActivationStackParams>()
 
+interface ActivationStep {
+  screenName: ActivationScreen
+  component: FunctionComponent
+}
+
+const activateProximityTracing: ActivationStep = {
+  screenName: ActivationScreens.ActivateProximityTracing,
+  component: ActivateProximityTracing,
+}
+
+const notificationPermissions: ActivationStep = {
+  screenName: ActivationScreens.NotificationPermissions,
+  component: NotificationPermissions,
+}
+
+const activationStepsIOS: ActivationStep[] = [
+  activateProximityTracing,
+  notificationPermissions,
+]
+
+const activationStepsAndroid: ActivationStep[] = [activateProximityTracing]
+
+const activationSteps =
+  Platform.OS === "ios" ? activationStepsIOS : activationStepsAndroid
+
+export const firstActivationScreen = activationSteps[0].screenName
+
 const ActivationStack: FunctionComponent = () => {
   const { t } = useTranslation()
   const navigation = useNavigation()
 
-  type ActivationStep =
-    | "ActivateProximityTracing"
-    | "NotificationPermissions"
-    | "AcceptEula"
-
-  const HeaderRight = (step: ActivationStep) => {
-    const determineStepText = () => {
-      const totalSteps = Platform.OS === "ios" ? 3 : 2
-
-      switch (step) {
-        case "AcceptEula":
-          return t("onboarding.step", { currentStep: 1, totalSteps })
-        case "ActivateProximityTracing":
-          return t("onboarding.step", { currentStep: 2, totalSteps })
-        case "NotificationPermissions":
-          return t("onboarding.step", { currentStep: 3, totalSteps })
-        default:
-          return t("onboarding.step", { currentStep: 1, totalSteps })
-      }
-    }
-
+  const HeaderRight = (currentStep: number, totalSteps: number) => {
     return (
       <View style={style.headerRight}>
         <GlobalText style={style.headerRightText}>
-          {determineStepText()}
+          {t("onboarding.step", { currentStep, totalSteps })}
```

Comment:
probably not important but should we have an `activation` namespace for this?

Reply:
agree, but also going to leave for a future commit.